### ISSUE TITLE
Validate for logic inside "describe", but outside of an actual test.

### ISSUE THREAD

**Issue by donaldpipowitch** [playwright-community/eslint-plugin-playwright#354]
Not sure if this is actually a Playwright bug or should be covered with an ESLint rule, but it looks like a tagged describe blog partially executes code that is outside of actual tests:

```ts
test.describe('Some describe...', { tag: ['@tag1', '@tag2'] },() => {
    useSomeLogic();

    // ...tests
});
```

If I use `--grep` with `@tag3` then `useSomeLogic();` seems to be executed. I have to move this function call inside the actual tests to filter it out correctly.
